github/repos_releases_test.go

```diff
@@ -44,15 +44,15 @@ func TestRepositoriesService_GetRelease(t *testing.T) {
 
 	mux.HandleFunc("/repos/o/r/releases/1", func(w http.ResponseWriter, r *http.Request) {
 		testMethod(t, r, "GET")
-		fmt.Fprint(w, `{"id":1}`)
+		fmt.Fprint(w, `{"id":1, "author": { "login": "authorname" }}`)
```

Comment:
Also, most other test data uses `"login":"l"` as login value, not "authorname". It doesn't matter much, but it's nice to be more consistent.